### PR TITLE
Update configuration.md with ASM steps

### PR DESCRIPTION
**Pull request by sourabhkatti**
Add ASM in the serverless configuration

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/serverless/installation/dotnet.md [hunk 1]
```diff
@@ -245,7 +245,7 @@ To install and configure the Datadog Serverless Plugin, follow these steps:
 {{< /tabs >}}
 
 ## What's next?
-
+- Turn on [threat monitoring][6] to get alerted on attackers targeting your service.
```
Comment (buraizu) [inline content/en/serverless/installation/dotnet.md:248]: Suggesting to move all the `What's next` section additions to somewhere after the `You can now...` or `Congratulations! You can now...` lines, as the implication is that this step is needed to view metrics, logs, and traces. Please let me know if I'm misunderstanding anything in this regard.

File: content/en/serverless/aws_lambda/configuration.md [hunk 2]
```diff
@@ -792,3 +828,4 @@ If you have trouble configuring your installations, set the environment variable
 [37]: /serverless/guide/extension_motivation/
 [38]: /serverless/guide#install-using-the-datadog-forwarder
 [39]: /serverless/guide/troubleshoot_serverless_monitoring/
+[40]: /serverless/installation#installation-instructions
```
Comment (buraizu) [inline content/en/serverless/aws_lambda/configuration.md:831]:
```suggestion
[40]: /serverless/installation#installation-instructions
[41]: https://app.datadoghq.com/security/appsec?column=time&order=desc
```

File: content/en/serverless/aws_lambda/configuration.md [hunk 3]
```diff
@@ -86,6 +89,39 @@ First, [install][1] Datadog Serverless Monitoring to begin collecting metrics, t
...
+   environment:
+     DD_EXPERIMENTAL_ENABLE_PROXY: true
+     AWS_LAMBDA_EXEC_WRAPPER: /opt/datadog_wrapper
+   ```
+
+Redeploy the function and invoke it. After a few minutes, it appears in [ASM views][3].
+
+[3]: https://app.datadoghq.com/security/appsec?column=time&order=desc
+
+To see Application Security Management threat detection in action, send known attack patterns to your application. For example, send an HTTP header with value `acunetix-product` to trigger a [security scanner attack][5] attempt:
```
Comment (buraizu) [inline content/en/serverless/aws_lambda/configuration.md:119]: This link is currently opening a blog post, which seems to match the other `[5]` reference link on line `223`. Is there a new link to go with `security scanner attack`?

File: content/en/serverless/aws_lambda/configuration.md [hunk 4]
```diff
@@ -86,6 +89,39 @@ First, [install][1] Datadog Serverless Monitoring to begin collecting metrics, t
...
+   ```sh
+   curl -H 'My-ASM-Test-Header: acunetix-product' https://your-function-url/existing-route
```
Comment (buraizu) [inline content/en/serverless/aws_lambda/configuration.md:121]:
```suggestion
   curl -H 'My-ASM-Test-Header: acunetix-product' https://<YOUR_FUNCTION_URL>/<EXISTING_ROUTE>
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for the PR, just flagging a few link issues and also made some suggestions per our [style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md).

**Comment by buraizu** [comment]
@sourabhkatti just reaching out to follow up on this PR -- I'm happy to take care of the remaining updates myself with your approval, but just want to confirm if this is the intended link for the `security scanner attack` text on line `119` of the  AWS Lambda configuration doc referenced in this [comment](https://github.com/DataDog/documentation/pull/18082#discussion_r1190299197):
https://docs.datadoghq.com/security/default_rules/security-scan-detected/

**Comment by sourabhkatti** [comment]
Hey Bryce, yes that's the correct link for this case. It refers the user to the https://docs.datadoghq.com/security/default_rules/security-scan-detected/ page to learn more about what this type of risk is.

**Comment by sourabhkatti** [comment]
Thanks for the help with bringing the PR up to the style guide also

**Review by buraizu — APPROVED** [review]
Thanks for your patience, LGTM

**Comment by buraizu** [comment]
@sourabhkatti thanks for confirming, I've updated the link and approved the PR. Just to confirm, is this PR ready to merge?

**Pull request merged**